inst/EpiModel2.rmd

```diff
@@ -17,30 +17,37 @@ suppressMessages(suppressWarnings(library(EpiModel)))
 
 The newest release of EpiModel, dubbed "EpiModel 2.0", brings with it several changes to the EpiModel workflow you may be accustomed to from previous version of EpiModel, as well as adding new functionality to the platform in order to make running dynamic infectious disease models even easier. In this tutorial, we will go through the major changes to EpiModel, detail the new features that have been added and in the final sections provide examples on how to migrate your code from EpiModel 1.x to EpiModel 2.0.
```

Comment:
```suggestion
The newest release of EpiModel, dubbed "EpiModel 2.0", brings with it several changes to the EpiModel workflow  as well as new functionality to the platform in order to make running dynamic infectious disease models even easier. In this tutorial, we will go through the major changes to EpiModel, detail the new features that have been added, and provide examples on how to migrate your code from EpiModel 1.x to EpiModel 2.0.
```

Reply:
@clchand23 I left this line in there as its meant to communicate to the previous users of EpiModel that they may find the changes documented worth noting.